src/lib/y2firewall/importer_strategies/suse_firewall.rb

```diff
@@ -101,18 +106,39 @@ def services(zone_name)
       # profile.
       #
       # @param zone_name [String]
+      # @return [Array<String>, nil]
       def interfaces(zone_name)
         interfaces = profile["FW_DEV_#{zone_name}"]
-        interfaces ? interfaces.split(" ") : nil
+        interfaces ? interfaces.split(" ").reject { |i| i == "any" } : nil
```

Comment:
Why not return always an `[]` or `nil` when no interfaces are found? I mean, it will return different values if interfaces are set to `any` (they will be filtered out anyway) and if no interfaces are found. Does it really matter?

It is not a big problem, as it is part of the private API, but still...

Reply:
Well, nil is returned when the key is not defined, in other case an array is returned.

Just needed to remove special cases.

Reply:
Then you should document that behaviour (or unify it). What about documenting this behaviour in the `@return` tag?